### ISSUE TITLE
conn is not defined

### ISSUE THREAD

**Issue by indeyets** [sidorares/node-mysql2#426]
Getting the following errors on recent version of mysql2:

```
/src/node_modules/mysql2/lib/connection.js:212
      conn._handleNetworkError(err);
      ^

ReferenceError: conn is not defined
    at /src/node_modules/mysql2/lib/connection.js:212:7
    at _combinedTickCallback (internal/process/next_tick.js:71:11)
    at process._tickDomainCallback (internal/process/next_tick.js:122:9)
```

the source shows that `conn` is not defined in that context, indeed

**Comment by sidorares** [comment]
yes, found this as well recently, will land together with #421 very soon. Thanks!

**Comment by indeyets** [comment]
@sidorares do you plan patch release with this fix? or should we revert to older version instead?

**Comment by sidorares** [comment]
@indeyets should be fixed in master, could you try it? I was going to push release today

**Comment by ReklatsMasters** [comment]
@sidorares Any updates? Can you push new release?

**Comment by sidorares** [comment]
@indeyets @ReklatsMasters sorry it took so long, published as v1.1.2
